Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -624,7 +664,7 @@ def cert_format(x):
 
 
 def urlscan_search_command(client):
-    LIMIT = int(demisto.args().get('limit'))
+    LIMIT = int(demisto.args().get('limit', 20))
```

Comment:
Please add a constant to the top of the page (perhaps with an explanation) instead of using magic numbers.